Personal_Accounts/src/test/java/org/openapitools/api_tests/aisp_accounts_accountId_balances_get.feature

```diff
@@ -14,10 +14,10 @@ Background:
   * def urlBase = karate.properties['url.base'] || karate.get('urlBase', 'http://localhost:8080')
   * url urlBase
   * def authToken = karate.properties['AUTH_TOKEN']
-  * configure headers = {Authorization: '#('Bearer ' + authToken)', 'x-fapi-auth-date': 'Sun, 10 Sep 2017 19:43:31 UTC', 'x-fapi-customer-ip-address': '169.254.169.254', 'x-fapi-interaction-id': '123456789', 'Accept-Language': 'en-US'}
+  * configure headers = {Authorization: '#(authToken)', 'x-fapi-auth-date': 'Sun, 10 Sep 2017 19:43:31 UTC', 'x-fapi-customer-ip-address': '169.254.169.254', 'x-fapi-interaction-id': '123456789', 'Accept-Language': 'en-US'}
```

Comment:
keyword `Bearer` cannot be added here, it should be included in the `AUTH_TOKEN` itself